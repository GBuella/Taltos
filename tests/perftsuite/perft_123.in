setboard n1n5/1Pk5/8/8/8/8/5Kp1/5N1N b - - 0 1
echo n1n5/1Pk5/8/8/8/8/5Kp1/5N1N b - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
