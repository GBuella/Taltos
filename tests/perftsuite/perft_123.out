n1n5/1Pk5/8/8/8/8/5Kp1/5N1N b - - 0 1
24
421
7421
124608
2193768
37665329
